proto/issuer/tx.proto

```diff
@@ -13,7 +13,7 @@ service Msg {
   rpc MintToken(MsgMintToken) returns (MsgMintTokenResponse);
 }
 
-// MsgCreateIssuer defines a SDK message for creating a new identifier.
+// MsgCreateIssuer defines a SDK message for creating a new emoney token issuer.
```

Comment:
```suggestion
// MsgCreateIssuer defines an SDK message for creating an emoney token issuer.
```

Reply:
use the phonetic sound for articles before acronyms (_a_ becomes _an_ since S sounds like ess), fun stuff, right? 

when we use "create" we don't need "new" because create and new are the same thing